### PR TITLE
Refactoring flotta agent scripts

### PR DESCRIPTION
**Pull request by bardielle**
* Using the bash file `flotta-image-builder-common-functions.sh` for common function
* Removing unused bash files
* Adding an iso image as part of flotta-agent-sti-upgrade script

### CODE REVIEW

File: flotta-image-builder-common-functions.sh [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+  dnf install -y jq isomd5sum genisoimage
+  composer-cli blueprints push $ISO_BLUEPRINT
+  BUILD_ID=$(composer-cli -j compose start-ostree --ref $REF --url http://$IMAGE_SERVER_ADDRESS/$IMAGE_NAME/repo/ edgedevice-iso edge-installer | jq '.build_id')
+  echo "waiting for build $BUILD_ID to be ready..."
+  while [ $(composer-cli -j compose status  | jq -r '.[] | select( .id == '$BUILD_ID' ).status') == "RUNNING" ] ; do sleep 5 ; done
+  if [ $(composer-cli -j compose status  | jq -r '.[] | select( .id == '$BUILD_ID' ).status') != "FINISHED" ] ; then
+      echo "image composition failed"
+      echo "check 'composer-cli compose status'"
+      exit 1
+  fi
```
Comment (masayag) [inline flotta-image-builder-common-functions.sh:190]: seems that this block is repeated several times and can be extracted into a common function.

**Review by masayag — COMMENTED** [review]
Nice work!
Pls let me know WDYT about the comment I had.
Also, did you get a chance to verify on both aarch and x86?

**Comment by bardielle** [comment]
@masayag Thank you for reviewing! 
I verified on x86 but I will verify on aarch and will update soon

**Review by pkliczewski — APPROVED** [review]

**Comment by pkliczewski** [comment]
> @masayag Thank you for reviewing! I verified on x86 but I will verify on aarch and will update soon

@bardielle any updates?

**Comment by bardielle** [comment]
Not yet
didn't forget about that

**Pull request merged**